apps/src/lib/kits/maker/ui/MakerStatusOverlay.jsx

```diff
@@ -64,42 +83,40 @@ const style = {
     padding: '2em',
     textAlign: 'center',
   },
+  footer: {
+    flex: '0 0 0',
```

Comment:
You're right - it was having issues, but changing to always use `flex-basis: auto` (which here looks like `flex: '0 0 auto',`) fixed things up.

Here it is working in IE11:
![image](https://cloud.githubusercontent.com/assets/1615761/24822264/a46b37c0-1ba8-11e7-96d5-44941353163e.png)